spec/unit/transaction/resource_harness_spec.rb

```diff
@@ -1,7 +1,7 @@
 #! /usr/bin/env ruby
 require 'spec_helper'
 
-require 'puppet/transaction/resource_harness'
```

Comment:
We need to preserve this require, otherwise, there's no guarantee `Puppet::Transaction::ResourceHarness` is defined.